app/pods/components/agenda/agenda-header/template.hbs

```diff
@@ -90,8 +90,11 @@
                           >
                             {{t "agenda-close"}}
                           </a>
-                        {{/if}}
-                      {{else}}
+                        </li>
+                        <li class="vlc-dropdown-menu__separator" aria-hidden="true"></li>
+                      {{/if}}
+                    {{else}}
+                      <li class="vlc-dropdown-menu__item">
                         <a href=""
```

Comment:
In geval van ontwerpagenda A waren er 2 separators te zien tussen acties.
Nu is er altijd max 1 separator tussen acties